dataeng/resources/warehouse-transforms-master-ci.sh

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env bash
+set -ex
+
+# Setup to run python script to create snowflake schema
+cd $WORKSPACE/analytics-tools/snowflake
+make requirements
+
+# Download Prod build manifest.json file from S3 and creating directory to place manifest file.
+cd $WORKSPACE/ && mkdir -p manifest
+
+pip install awscli
+
+aws s3 cp s3://edx-dbt-docs/manifest.json ${WORKSPACE}/manifest
+
+# Setup to run dbt commands
+cd $WORKSPACE/warehouse-transforms
+
+PREV_COMMIT_ID=$(git rev-parse @~)
+
+git diff $PREV_COMMIT_ID --name-only
```

Comment:
compare difference between the HEAD and second last commit to find out the projects files have changed.